op-chain-ops/cmd/registry-data/main.go

```diff
@@ -91,17 +91,19 @@ type GenesisAccount struct {
 }
 
 type Genesis struct {
-	Nonce      uint64         `json:"nonce"`
-	Timestamp  uint64         `json:"timestamp"`
-	ExtraData  []byte         `json:"extraData"`
-	GasLimit   uint64         `json:"gasLimit"`
-	Difficulty *hexutil.Big   `json:"difficulty"`
-	Mixhash    common.Hash    `json:"mixHash"`
-	Coinbase   common.Address `json:"coinbase"`
-	Number     uint64         `json:"number"`
-	GasUsed    uint64         `json:"gasUsed"`
-	ParentHash common.Hash    `json:"parentHash"`
-	BaseFee    *hexutil.Big   `json:"baseFeePerGas"`
+	Nonce         uint64         `json:"nonce"`
+	Timestamp     uint64         `json:"timestamp"`
+	ExtraData     []byte         `json:"extraData"`
+	GasLimit      uint64         `json:"gasLimit"`
+	Difficulty    *hexutil.Big   `json:"difficulty"`
+	Mixhash       common.Hash    `json:"mixHash"`
+	Coinbase      common.Address `json:"coinbase"`
+	Number        uint64         `json:"number"`
+	GasUsed       uint64         `json:"gasUsed"`
+	ParentHash    common.Hash    `json:"parentHash"`
+	BaseFee       *hexutil.Big   `json:"baseFeePerGas"`
+	ExcessBlobGas *uint64        `json:"excessBlobGas"` // EIP-4844
+	BlobGasUsed   *uint64        `json:"blobGasUsed"`   // EIP-4844
```

Comment:
we don't want `,omitempty` here?

Reply:
I'm not sure -- are you thinking to somehow make this backwards compatible for new chains not activating Ecotone at Genesis? I think generally we expect these fields to be present given the other changes in this PR.

Reply:
ok, we can always adapt if it turns out we need it 👍🏻